sqrl-execute/sqrl-execute-http/sqrl-execute-http-vertx/src/main/java/com/datasqrl/graphql/config/CorsHandlerOptionsConverter.java

```diff
@@ -0,0 +1,80 @@
+package com.datasqrl.graphql.config;
+
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.JsonObject;
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CorsHandlerOptionsConverter {
+
```

Comment:
Why can we not use Jackson for the mapping?

Reply:
This is idiomatic for vertx. This really should be the same configuration pattern as the rest of vertx, not sure why it wasn't.